pkg/datasource/sample-datasource.go

```diff
@@ -33,10 +33,7 @@ func (td *SampleDatasource) QueryData(ctx context.Context, req *backend.QueryDat
 
 	// loop over queries and execute them individually.
 	for _, q := range req.Queries {
-		res, err := td.query(ctx, q)
-		if err != nil {
```

Comment:
I wanted to show that you can err in different ways. But that might not be suitable for tutorials/boilerplate.

Reply:
Understood, could move it into QueryData as per initial comment on PR.